hathor/transaction/base_transaction.py

```diff
@@ -638,7 +638,12 @@ def reset_metadata(self) -> None:
 
         self.storage.save_transaction(self, only_metadata=True)
 
-    def update_accumulated_weight(self, *, stop_value: float = inf, save_file: bool = True) -> TransactionMetadata:
+    def update_accumulated_weight(
+        self,
+        *,
+        stop_value: float | None = None,
```

Comment:
I'd change it to work, i.e., `stop_value: int | None`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1193/commits/f42c4a7e07adfbdd7a8fce8fb73a488a2a5fa8f2

Reply:
Thanks!